vector/src/main/res/xml/vector_settings_labs.xml

```diff
@@ -68,4 +69,11 @@
         android:key="SETTINGS_LABS_RENDER_LOCATIONS_IN_TIMELINE"
         android:title="@string/labs_render_locations_in_timeline" />
 
-</androidx.preference.PreferenceScreen>
\ No newline at end of file
+    <im.vector.app.core.preference.VectorSwitchPreference
+        android:defaultValue="@bool/settings_labs_show_live_sender_info_default"
+        app:isPreferenceVisible="@bool/settings_labs_show_live_sender_info_visible"
+        android:key="SETTINGS_LABS_SHOW_LIVE_SENDER_INFO"
+        android:summary="@string/labs_show_latest_profile_description"
+        android:title="@string/labs_show_latest_profile" />
```

Comment:
Sorry, I did not notice that earlier, but I think this setting could be moved to `Settings/Preferences`, `Timeline` section, maybe just below `Show Message Bubble`.
I do not know what other clients are doing, and it can be moved later, this is not a big deal!